### DISCUSSION TITLE
HTTPS and image download

### DISCUSSION THREAD

**Discussion by Marin-Kitagawa** [qutebrowser/qutebrowser#6514] [Q&A] [answered]
1. How to make `qutebrowser` use `HTTPS` by default? If I open only a part of a website, say `reddit.com`, it first opens in `http` and then changes to `https`. How to make make `qutebrowser` to use `HTTPS` by default? 
2. When I open an image and only the image shows in the webpage, for example, in this URL `https://i.redd.it/9lirkoh226371.png` how to dowload the image from keybinding(s), command(s) or both i.e. without right-clicking and saving the image? Also, is(are) there any keybinding(s) or command(s) available for downloading the webpage?

**Comment by The-Compiler** [comment, marked as answer]
1. There's `:open -s` (which you can change bindings to). Note that it'll always force a `https://` scheme, even if you explicitly enter `http://`, as qutebrowser currently can't distinguish the two. Also note that this is pretty common behavior with other browsers as well. I think there was an issue somewhere about making `-s` more intelligent, but I can't find it right now.
2. `gd` (`:download`) downloads the current web page.

**Reply by Marin-Kitagawa** [reply]
Thanks for that
